tests/manual-test-cases/Group5-Functional-Tests/5-4-High-Availability.robot

```diff
@@ -232,15 +232,18 @@ Test
     \     Should Be Equal As Integers  ${rc}  0
     \     Should Be Equal  ${output}  running
     \     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} rm -f ${c}
-    \     Should Be Equal As Integers  ${rc}  0
```

Comment:
I'm not sure if removing this is accurate - this likely invalidates the intent of the test since it is now possible for the test to pass without `rm -f` succeeding. With this change, the test will pass if `rm -f`:

1) returns 0 (valid)
2) fails with exit code other than 0 and the output didn't contain `ESX host is temporarily disconnected.`

Is 2. still a valid test scenario wherein the test should pass?

Reply:
Good point - had missed this when I read it over.